seungjong/routes/postRouter.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const router = express.Router();
+const postController = require('../controllers/postController');
```

Comment:
```suggestion
const express = require('express');

const postController = require('../controllers/postController');

const router = express.Router();
```
이 부분외에 다른 파일들도 모두 적용시켜주세요.